.github/workflows/ci.yml

```diff
@@ -185,7 +185,7 @@ jobs:
               run: vendor/bin/phpunit --colors=always
 
             - name: Run the functional tests
-              run: vendor/bin/phpunit --testsuite=functional --colors=always
+              run: SYMFONY_DEPRECATION_HELPER='max[self]=0&amp;baselineFile=./.github/deprecations.json' vendor/bin/phpunit --testsuite=functional --colors=always
```

Comment:
I think we can remove the `deprecations.json` file altogether. The variable is called `SYMFONY_DEPRECATIONS_HELPER` and not `SYMFONY_DEPRECATION_HELPER`, which means that the file was never read (check the actions log). 😄 

And the PHP 8.1 test ist green now anyway. 🎉

Reply:
This is super weird, because without it (specified in the `phpunit.xml.dist` like before) it failed in my local 8.1 container yesterday. 

I'll rerun it and then update the PR accordingly.

Reply:
The `Unsilenced deprecation notices` now still show red locally. BUT: the error code is actually `0`. I don't get it. :exploding_head: :question: 

So, nevermind, the baseline file is gone in d8a15fcf3.

Reply:
These are not self deprecations, therefore they do not make the build fail.